setup.py

```diff
@@ -42,6 +42,7 @@
     'pytest-cov>=1.8.0',
     'pytest-pep8>=1.0.6',
     'pytest>=2.8.0',
+    'responses>=0.5.1'
```

Comment:
Normally we've been using HTTPretty for request mocking - any particular reason for using ``responses`` library?